### PR TITLE
Add NONNC keyword to the SimulationConfig

### PR DESCRIPTION
**Pull request by lisajulia**
@blattms Can you have a look?

### CODE REVIEW

**Comment by blattms** [comment]
Looks good to me. Nice that you also changed the python bindings.

We should probably add a unit test for hasNONNC to e,g. tests/parser/SimulationConfigTests.cpp, though. Not sure where to test the python bindings.

File: src/opm/input/eclipse/EclipseState/SimulationConfig/SimulationConfig.cpp [hunk 1]
```diff
@@ -113,6 +123,7 @@ namespace Opm {
         result.m_bcconfig = BCConfig::serializationTestObject();
         result.m_rock_config = RockConfig::serializationTestObject();
         result.m_useCPR = false;
+        result.m_useNONNC = false;
```
Comment (akva2) [inline src/opm/input/eclipse/EclipseState/SimulationConfig/SimulationConfig.cpp:126]: this should be true - it should not be the default value as a missing serialization might slip through the cracks then (general rule, i of course see that you haven't forgotten it here..).

Comment (lisajulia) [reply src/opm/input/eclipse/EclipseState/SimulationConfig/SimulationConfig.cpp]: ok :) I changed it to `result.m_useNONNC = true;`

**Comment by blattms** [comment]
jenkins build this opm-simulators=5134

**Comment by blattms** [comment]
jenkins build this please

**Pull request merged**